cvat/apps/dataset_manager/tests/test_formats.py

```diff
@@ -496,6 +511,21 @@ def test_frames_outside_are_not_generated(self):
             self.assertTrue(frame.frame in range(6, 10))
         self.assertEqual(i + 1, 4)
 
+    def test_can_import_annotations_for_image_with_dots_in_filename(self):
+        export_formats = [f.DISPLAY_NAME for f in dm.views.get_export_formats()]
+        formats = [f.DISPLAY_NAME for f in dm.views.get_import_formats() if
+            f.DISPLAY_NAME in export_formats]
+        for format_name in formats:
+            if format_name == "VGGFace2 1.0":
+                self.skipTest("Format is disabled")
```

Comment:
Why this format is disabled?

Reply:
I skipped the test for this format because VGGFace does not support annotations that are generated in the `_generate_annotations(...)` function. Same problem with ICDAR formats. I can specify annotations for these formats, but this will lead to a lot of code. So I would prefer to remove this test and include images with dots in filename in this test https://github.com/openvinotoolkit/cvat/blob/4ceb5da2447805c29ca4e20108bb2a2fb31ca6d3/cvat/apps/engine/tests/test_rest_api.py#L2966

Reply:
Actually, the real problem was that the reason of skipping was not described in tests. Thus, a reader can't judge, whether this test already can be enabled or not.